src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,52 @@
+import "./footer.css";
+
+function Footer() {
+  return (
+    <footer>
+      <ul class="footer_list_items">
+        <li>
+          <a class="mr-1 footer_link inherit-clr" href="/about">
+            About Us
+          </a>
+        </li>
+        <li>
+          <a class="mr-1 ml-1 inherit-clr" href="/customer-service">
+            Customer Service
+          </a>
+        </li>
+        <li>
+          <a class="mr-1 ml-1 inherit-clr" href="/subscription">
+            Subscription
+          </a>
+        </li>
+        <li>
+          <a class="mr-1 ml-1 inherit-clr" href="/contact-us">
+            Contact Us
+          </a>
+        </li>
+        <li>
+          <a class="mr-1 ml-1 inherit-clr" href="/category">
+            Category
+          </a>
+        </li>
+        <li>
+          <a class="mr-1 ml-1 inherit-clr" href="./productListing.html">
+            All Product
+          </a>
+        </li>
+        <li>
+          <a class="mr-1 ml-1 inherit-clr" href="./cartManagement.html">
```

Comment:
try to use react router and replace anchor tag with Link tag

Reply:
Yaah I was using that , but i getting some npm packages version conflict, which has not resolved thats why i am not using it yet, I will use once I resolve that with MA TA